test/integration/integration/utils/http/http.go

```diff
@@ -383,11 +383,21 @@ func GetTestToken(t *testing.T, gwAddr string, scopes ...string) string {
 	}
 	resp, err := client.Do(req)
 	if err != nil {
-		t.Fatalf("failed to get token: %v", err)
+		t.Logf("failed to get token: %v retrying after 20s ...", err)
```

Comment:
Is this to wait for enforcer test token endpoint to become available?

Reply:
yes, this only happen once in the very first test case, after that this flow doesn't hit